hibike/hibike_message.py

```diff
@@ -112,31 +114,152 @@ def checksum(data):
 def send(serial_conn, message):
   m_buff = message.toByte()
   chk = checksum(m_buff)
-  m_buff.append(chr(chk))
+  m_buff.append(chk)
   encoded = cobs_encode(m_buff)
   out_buf = bytearray([0x00, len(encoded)]) + encoded
```

Comment:
I could rename 0 to PACKET_DELIMITER_BYTE or something but I'm not sure if that makes this easier to read